internal/image/validate.go

```diff
@@ -18,8 +18,14 @@ package image
 
 import (
 	"context"
+	"encoding/json"
+	"sort"
+	"time"
 
 	conftestOutput "github.com/open-policy-agent/conftest/output"
+	"github.com/qri-io/jsonpointer"
```

Comment:
This library seems to be unmaintained? Last commit almost two years ago.

Reply:
Seems good enough for what we need, we're pulling it via [qri-io/jsonschema](https://github.com/qri-io/jsonschema) as well, which serves us well. I like that it has no additional dependencies...